supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -874,6 +885,16 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 
 *See also*:
 
+[[rolling-stream]]
+==== image:images/yes.png[yes] Rolling stream (noun)
+*Description*: _Rolling Application Streams_ include RHEL tools and applications that are updated frequently. Later versions of these streams replace earlier versions. Therefore, only one version is supported at a time.
```

Comment:
Should "Application" be removed to be consistent with the term? or should it be included in the term? Also the capitalization is different for "Stream" - not sure what is right

Reply:
Thank you, I checked with the sst writer that the terms are interchangeable. Rewording this a bit.